coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -117,6 +118,7 @@ func TestHeartbeat(t *testing.T) {
 	//nolint:dogsled // ｡:ﾟ૮ ˶ˆ ﻌ ˆ˶ ა ﾟ:｡
```

Comment:
wtf is ｡:ﾟ૮ ˶ˆ ﻌ ˆ˶ ა ﾟ:｡

Reply:
The result of late-night hacking and looking ASCII dogs.